modules/post/multi/recon/local_exploit_suggester.rb

```diff
@@ -119,33 +121,38 @@ def run
     @local_exploits.each do |m|
       begin
         checkcode = m.check
+      rescue Rex::Post::Meterpreter::RequestError => e
```

Comment:
Out of interest, what's the rationale behind special casing this exception in particular? 👀

Reply:
I know it was there previously, but was it investigated in more detail whether it's needed still? 🕵️

Reply:
If I'm not confuddled, it's that this error indicates the session is borked and that you can't run any more checks, where as any other error indicates it was just the check that had an issue

Reply:
It's raised in other circumstances too:
```
meterpreter > run post/windows/manage/migrate

[*] Running module against HOME-F006222D6C
[*] Current server process: webappmon.exe (7064)
[*] Spawning notepad.exe process to migrate to
[-] Post failed: Rex::Post::Meterpreter::RequestError stdapi_sys_process_execute: Operation failed: Access is denied.
[-] Call stack:
[-]   /projects/exploiting/trunk/lib/rex/post/meterpreter/extensions/stdapi/sys/process.rb:163:in `execute'
[-]   (eval):80:in `create_temp_proc'
[-]   (eval):49:in `run'
=end
```